README.md

```diff
@@ -1,14 +1,41 @@
-# More Information
+# Overview
 
-You have a new Teams project scaffolded! To understand more about the structure of the project, you can read the readme files listed below to get further information.
+At current stage, Teams App has a wealth of features and many applications. At present, however, only the web documents provide information on these features. This way has a weak direct-viewing display effect. Therefore, I decided to complete a Teams App to show our Teams Extension (Teams Toolkit) features. My goal is to allow developers of Teams App to easily access it and get a better development experience.
```

Comment:
Tips: do not use "I" for repo description

Reply:
Thanks, I will modify it asap.